src/vs/workbench/browser/parts/editor/editorParts.ts

```diff
@@ -93,6 +93,7 @@ export class EditorParts extends Disposable implements IEditorGroupsService, IEd
 		disposables.add(part.onDidRemoveGroup(group => this._onDidRemoveGroup.fire(group)));
 		disposables.add(part.onDidMoveGroup(group => this._onDidMoveGroup.fire(group)));
 		disposables.add(part.onDidActivateGroup(group => this._onDidActivateGroup.fire(group)));
+		disposables.add(part.onDidChangeGroupMaximized(group => this._onDidChangeGroupMaximized.fire(group)));
 
```

Comment:
I changed the event to not pass the group anymore as it is never used. However, I am not sure about here. Should we still pass the group through all layers to add it here or maybe just indicate which editorPart is in "maximized view mode"?

Reply:
Good point. I am fine removing the editor group from the event, but then I think the associated methods to figure out if a group is maximized or not in an editor part should also not require me to send in a group, so that the change is symmetric.

For the event itself: we probably need some kind of indication which editor part has a group maximized, so then I would have `IEditorPart` in the event as property. Otherwise a listener would not be able to tell which editor part in which widow is impacted.

Reply:
Currently not possible with the way EditorPart and EditorParts share the interface. To be discussed...